doc/releases/pacific.rst

```diff
@@ -5,6 +5,182 @@ Pacific
 Pacific is the 16th stable release of Ceph.  It is named after the
 giant pacific octopus (Enteroctopus dofleini).
 
+v16.2.5 Pacific
+===============
+
+This is the fifth  backport release in the Pacific series.  We recommend all users
+update to this release.
+
+Notable Changes
+--------------
```

Comment:
Missing a dash or two here.

Reply:
@idryomov I don't know why :(

@djgalloway  can you take a look ?  does the output need lot of editing after the script ran ?

Reply:
Yes.  The missing dash in the header is a manual process though.  Unrelated to `ceph-release-notes` output.